README.md

```diff
@@ -359,8 +359,12 @@ const columns: ColumnDef<Person>[] = [
   // ...other columns
   {
     id: 'settings_column_id',
-    header: (context) => <TableSettings {...context} columnId="settings_column_id" />,
-  }, // or u can use function getSettingsColumn
+    header: ({table}) => <TableSettings table={table} />,
+    meta: {
+      hideInSettings: false, // Optional. Allows to hide this column from settings popover
+      inSettings: 'some string or any html', // Optional. Overrides header field for settings popover (if you need different content for header and settings popover)
```

Comment:
> `any html`

Do you mean ReactNode? Because html is escaped by react

Reply:
changed to ReactNode